src/go.mod

```diff
@@ -3,17 +3,25 @@ module github.com/containers/toolbox
 go 1.13
 
 require (
-	github.com/HarryMichal/go-version v1.0.0
+	github.com/HarryMichal/go-version v1.0.1
 	github.com/acobaugh/osrelease v0.0.0-20181218015638-a93a0a55a249
 	github.com/briandowns/spinner v1.10.0
 	github.com/docker/go-units v0.4.0
-	github.com/fsnotify/fsnotify v1.4.7
-	github.com/godbus/dbus/v5 v5.0.3
-	github.com/mattn/go-isatty v0.0.8
+	github.com/fsnotify/fsnotify v1.4.9
+	github.com/godbus/dbus/v5 v5.0.6
+	github.com/konsorten/go-windows-terminal-sequences v1.0.3 // indirect
```

Comment:
We don't need to highlight the changes in the indirect dependencies in the commit message. There are just too many of those and they keep coming and going. It's better to only highlight our direct dependencies in the commit message, unless there's one that we really need to be careful about.

Reply:
Fair enough.